dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/SingleDataRequestManager.kt

```diff
@@ -23,7 +24,7 @@ import org.springframework.transaction.annotation.Transactional
 class SingleDataRequestManager(
     @Autowired private val dataRequestLogger: DataRequestLogger,
     @Autowired private val companyApi: CompanyDataControllerApi,
-    @Autowired private val singleDataRequestEmailSender: SingleDataRequestEmailSender,
+    @Autowired private val singleDataRequestEmailMessageSender: SingleDataRequestEmailMessageSender,
     @Autowired private val utils: DataRequestProcessingUtils,
 ) {
```

Comment:
Warum wird die folgende Zeile in der process-Funktion eigentlich doppelt aufgerufen:

`dataRequestLogger.logMessageForReceivingSingleDataRequest(singleDataRequest.companyIdentifier)`

Reply:
Das weiß keiner so genau Passiert jetzt aber nicht mehr.